lib/Doctrine/ORM/Query.php

```diff
@@ -196,7 +201,7 @@ class Query extends AbstractQuery
      */
     public function getSQL()
     {
-        return $this->parse()->getSqlExecutor()->getSqlStatements();
+        return $this->getSqlExecutor()->getSqlStatements();
```

Comment:
I would ignore this one, but it would be worth attempting to make the phpdoc of `getSqlStatements()` more precise on 2.18.x

Reply:
https://github.com/doctrine/orm/pull/11195